linuxrc_hostname.md

```diff
@@ -1,30 +1,35 @@
 # hostname setting in installation/rescue system
 
-There are 3 cases:
+The hostname can be set (in order of precedence):
 
-1. no hostname is set
+1. via `hostname` boot option
+2. via DHCP (either from HOSTNAME or FQDN)
+3. Default hostname `install`
 
-2. hostname is set via dhcp (either from HOSTNAME or FQDN)
+linuxrc defaults to `install`, which may be overridden by DHCP(2) or user(1).
+(1) takes precedence over (2).
 
-3. hostname is set via `hostname` boot option
-
-linuxrc defaults to `install`, which may be overridden by dhcp (2) or user
-(3). (3) takes precedence over (2).
 
 ## installation system
 
-When yast is started, the value from (3) is passed to yast via
-`/etc/install.inf::Hostname` and ends up in `/etc/hostname` of the installed system.
+When yast is started, the value from (1) is passed to yast via
+`/etc/install.inf::Hostname` and ends up in `/etc/hostname` of the installed
+system.
+
+When linuxrc doesn't set `/etc/install.inf::Hostname`, yast generates a default
+`/etc/hostname` entry:
+- Before SLE12, of the form `linux-XXXX`
+- Since SLE12, `linux-XXXX.suse`
+- Since SLE15-SP2, `linux`
```

Comment:
YaST should not propose a hostname if no one was configured explicitly according to documentation

Reply:
But, it does, because `SetHostname=1` by default. I expressed it wrong.
WDYT about the new wording?